packages/provider/package.json

```diff
@@ -5,12 +5,19 @@
     "license": "Apache-2.0",
     "main": "./dist/index.js",
     "type": "module",
+    "exports": {
+        ".": {
+            "import": "./dist/index.js",
+            "require": "./dist/cjs/index.cjs"
+        }
+    },
     "scripts": {
         "clean": "tsc --build --clean",
-        "build": "tsc --build --verbose",
+        "build": "tsc --build --verbose tsconfig.json",
+        "build:cjs": "npx vite --config vite.cjs.config.ts build",
         "build:debug": "tsc --build --verbose",
         "build:test": "tsc --build --verbose tsconfig.test.json",
-        "build:config": "tsc --project",
+        "build:config": "tsc -project",
```

Comment:
are you sure this is the cmd? my tsc help says:
```
 --project, -p  Compile the project given the path to its configuration file, o                 r to a folder with a 'tsconfig.json'.

```

Reply:
yeah this is wrong